docs/CLI/5_developer_guide.md

```diff
@@ -105,22 +109,19 @@ To update or add new dependencies, run `go get <package name>`.
     Git commit:     173c010570c48e4aa18ce186cae8cc812f8e8b6e
     Git tree state: clean
     ```
-14. Create a PR on [fishworks/fish-food](https://github.com/fishworks/fish-food) with the new ORAS version and checksums. This can be done by navigating to [this file](https://github.com/fishworks/fish-food/blob/main/Food/oras.lua) in your browser, and clicking the "Edit" button, making the changes and clicking "Propose changes", which will allow you to open a PR. You can also do this in your terminal/editor, of course.
```

Comment:
Removed since the tool is archived, see https://github.com/fishworks/fish-food#this-project-is-being-archived